packages/server/src/restaurants/api/RestaurantDto.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiProperty, PickType } from '@nestjs/swagger';
+
+import { ApiObjectIdProperty } from '../../shared';
+
+class RestaurantDto {
```

Comment:
W zależności od tego jak mają wyglądać dalsze endpointy tu może brakować kilku pól ale to IMO można dodać w PR związanych bezpośrednio z restauracjami.